SlaOptions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DependencyFlow
+{
+    public class SlaOptions
+    {
+        public Dictionary<string, Sla> Repositories { get; set;  }
```

Comment:
```suggestion
        public IDictionary<string, Sla>? Repositories { get; set;  }
```

nits: `IDictionary` for public properties and it's not initialized so it gets to be nullable ;). Or you could initialize it to an empty dictionary, or a dictionary containing only a hard-coded `[Default]` value.

Reply:
Fixed